blog/2024-07-07-openhab-4-2-release.md

```diff
@@ -0,0 +1,198 @@
+
+To allow storing multiple values to persistence in one go, Persistence Extensions methods now support working with TimeSeries instead of individual values when persisting or retrieving data.
+Time Series are series of time/value pairs, and can be constructed in rules, e.g. by modifying a Time Series retrieved from another persistence service or another Item.
+
+Also, there was no way before to retrieve the time of the last state change for an item from persistence.
+The existing methods (`previousState()` or `previousState(true)`) would not give the right result if the same value had been persisted multiple times.
+A new method `lastChange()` has been added to resolve this.
+
+Scripting languages (Rules DSL, JavaScript, jRuby and Blockly) have implement full support of these new and enhanced Persistence Extensions, please refer to the individual languages' documentation for more information.
+
+As much has changed, it is worthwhile to read the [documentation](/docs/configuration/persistence.html#persistence-extensions-in-scripts-and-rules) to update your knowledge about openHAB persistence extensions and learn everything about the new features.
+
+## Main UI Enhancements
+
+_Florian Hotze, openHAB Maintainer_
+
+### Dependency & Build System Upgrades
+
+openHAB 4.2 upgrades most of Main UI's dependencies to their latest version, including the charting engine [Apache ECharts](https://echarts.apache.org) and the date and time manipulation library [dayjs](https://day.js.org).
+As Main UI is still using Vue 2, we would like to upgrade to Vue 3 in the future - by upgrading to the latest Vue 2.x version, we have taken a minor step towards Vue 3.
+
+In addition to our runtime dependencies, we have also upgraded our build system to the latest tooling: Webpack 5 and Node.js 20.
+This ensures we can always use the latest Webpack features when bundling the Main UI source code, and moves away from Node.js 16, which is now EOL.
+
+Many thanks to Miguel Álvarez ([@GiviMAD](https://github.com/givimad)) for the Webpack upgrade!
+
+### Semantic Model Templates
+
+@justingeorgi: https://github.com/openhab/openhab-webui/pull/2105
+
+### Persistence Settings Menu
+
+Persistence configuration through the UI has been [supported since openHAB 4.0](/blog/2023-07-23-openhab-4-0-release.html#persistence-configuration), but unfortunately it was hidden in the add-on settings and difficult to find.
+
+This finally changes with openHAB 4.2 with the introduction of the new persistence settings menu:
+
+<p align="center"><img style="max-width: 70%;" src="/uploads/2024-07-07-openhab-4-2-release/persistence-menu.png"/></p>
+
+The new page is reachable through the settings menu and allows to configure the default persistence service, persistence policies and installation of more persistence add-ons.
+
+Thanks to Mark Herwege ([@mherwege](https://github.com/mherwege)) for the contribution!
+
+### Item Form Enhancements
+
+@jimtng
```

Comment:
Items are one of the core elements in an openHAB system.
To make viewing and editing them easier and more streamlined, the Item Form has received some minor cosmetic and functional improvements.
For dimensioned items, it is now easier to set the item's unit directly on the form with a default unit pre-filled.
Managing item's parent groups should now be easier too.